### PR TITLE
Simplify Enterprise Integrator Dockerfiles

### LINKED ISSUE #12: Simplify Enterprise Integrator Dockerfile
**Description:**
The existing WSO2 Dockerfiles use a complex set of bash scripts and Puppet for building the Docker images. These bash scripts have been used for improving certain aspects of Docker image build process and Puppet has been used for configuration management. Nevertheless, with our experience and the feedback received from our users we found that it would be much better to have plain Dockerfiles for building WSO2 Docker images than incorporating such features.

Above approach has already been followed in kubernetes-apim repository for building API Manager Docker images. We can take that as the baseline and update Enterprise Integrator Dockerfile accordingly.

**Affected Product Version:**
Enterprise Integrator 6.1.1

**Related Issues:**
https://github.com/wso2/docker-is/issues/15, https://github.com/wso2/docker-apim/issues/61, https://github.com/wso2/docker-das/issues/1

### PR DESCRIPTION
**Pull request by DilanUA**
## Purpose
The existing WSO2 Dockerfiles use a complex set of bash scripts and Puppet for building the Docker images. These bash scripts have been used for improving certain aspects of Docker image build process and Puppet has been used for configuration management. Nevertheless, with our experience and the feedback received from our users we found that it would be much better to have plain Dockerfiles for building WSO2 Docker images than incorporating such features.

Above approach has already been followed in kubernetes-apim repository for building API Manager Docker images. We have taken that as the baseline here and updated Enterprise Integrator Dockerfiles accordingly.

## Goals
This includes replacing the existing docker image build process of docker-ei with newly proposed plain dockerfile based image build approach. This resolves #12.

### CODE REVIEW

File: dockerfile/README.md [hunk 1]
```diff
@@ -1,59 +1,94 @@
-# Dockerfile for WSO2 Enterprise Integrator #
-The Dockerfile defines the resources and instructions to build the Docker images with the WSO2 products and runtime configurations.
+# Dockerfiles for WSO2 Enterprise Integrator #
+This section defines dockerfiles and step-by-step instructions to build docker images for multiple run-times <br>
+provided by WSO2 Enterprise Integrator 6.1.1, namely : <br>
+1. Integrator
+2. Business-process
+3. Broker
+4. Msf4j
```
Comment (imesh) [inline dockerfile/README.md]: It might be better to write MSF4J in uppercase.

File: dockerfile/README.md [hunk 2]
```diff
@@ -1,59 +1,94 @@
-# Dockerfile for WSO2 Enterprise Integrator #
-The Dockerfile defines the resources and instructions to build the Docker images with the WSO2 products and runtime configurations.
+# Dockerfiles for WSO2 Enterprise Integrator #
+This section defines dockerfiles and step-by-step instructions to build docker images for multiple run-times <br>
+provided by WSO2 Enterprise Integrator 6.1.1, namely : <br>
+1. Integrator
+2. Business-process
```
Comment (imesh) [inline dockerfile/README.md]: Might be better to refine as "Business Process"

File: dockerfile/README.md [hunk 3]
```diff
@@ -1,59 +1,94 @@
-# Dockerfile for WSO2 Enterprise Integrator #
-The Dockerfile defines the resources and instructions to build the Docker images with the WSO2 products and runtime configurations.
+# Dockerfiles for WSO2 Enterprise Integrator #
+This section defines dockerfiles and step-by-step instructions to build docker images for multiple run-times <br>
```
Comment (imesh) [inline dockerfile/README.md]: Might be better to write the word runtime without a dash. WDYT?

File: dockerfile/integrator/Dockerfile [hunk 4]
```diff
@@ -14,16 +13,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-
+#
 # ------------------------------------------------------------------------
```
Comment (imesh) [inline dockerfile/integrator/Dockerfile]: This license header seems to be slightly different from the rest of the license headers.

File: dockerfile/README.md (hunk 1 above)
Comment (DilanUA) [reply dockerfile/README.md]: Make sense. Will update. Thanks.

File: dockerfile/README.md (hunk 2 above)
Comment (DilanUA) [reply dockerfile/README.md]: Make sense. Will update. Thanks.

File: dockerfile/README.md (hunk 3 above)
Comment (DilanUA) [reply dockerfile/README.md]: Agreed. Will update. Thanks.

File: dockerfile/integrator/Dockerfile (hunk 4 above)
Comment (DilanUA) [reply dockerfile/integrator/Dockerfile]: We have kept the same style on all the recent product-wide refinements made to dockerfiles. Thus this change.

**Pull request merged**